### PR TITLE
Change `border-style` to <line-style> 

### PR DESCRIPTION
**Pull request by estelle**
The enumerated data type is `<line-style>` not `border-style` or `<border-style>`. When we have the new `<line-style>` page we can change all these links.

https://github.com/mdn/content/pull/26487

### CODE REVIEW

File: files/en-us/web/css/border-block-end/index.md [hunk 1]
```diff
@@ -44,7 +44,7 @@ The `border-block-end` is specified with one or more of the following, in any or
 
 - `<'border-width'>`
   - : The width of the border. See {{cssxref("border-width")}}.
-- `<'border-style'>`
+- {{cssxref("line-style")}}
   - : The line style of the border. See {{cssxref("border-style")}}.
```
Comment (dipikabh) [inline files/en-us/web/css/border-block-end/index.md:48]:
```suggestion
- `<'border-style'>`
  - : The line style of the border. See {{cssxref("line-style")}}.
```

File: files/en-us/web/css/border-block/index.md [hunk 2]
```diff
@@ -44,7 +44,7 @@ The `border-block` is specified with one or more of the following, in any order:
 
 - `<'border-width'>`
   - : The width of the border. See {{cssxref("border-width")}}.
-- `<'border-style'>`
+- {{cssxref("line-style")}}
   - : The line style of the border. See {{cssxref("border-style")}}.
```
Comment (dipikabh) [inline files/en-us/web/css/border-block/index.md:48]:
```suggestion
- `<'border-style'>`
  - : The line style of the border. See {{cssxref("line-style")}}.
```

File: files/en-us/web/css/border-inline-end/index.md [hunk 3]
```diff
@@ -44,7 +44,7 @@ The `border-inline-end` is specified with one or more of the following, in any o
 
 - `<'border-width'>`
   - : The width of the border. See {{cssxref("border-width")}}.
-- `<'border-style'>`
+- {{cssxref("line-style")}}
   - : The line style of the border. See {{cssxref("border-style")}}.
```
Comment (dipikabh) [inline files/en-us/web/css/border-inline-end/index.md:48]:
```suggestion
  - : The line style of the border. See {{cssxref("line-style")}}.
```

File: files/en-us/web/css/border-inline-start/index.md [hunk 4]
```diff
@@ -44,7 +44,7 @@ The `border-inline-start` is specified with one or more of the following, in any
 
 - `<'border-width'>`
   - : The width of the border. See {{cssxref("border-width")}}.
-- `<'border-style'>`
+- {{cssxref("line-style")}}
   - : The line style of the border. See {{cssxref("border-style")}}.
```
Comment (dipikabh) [inline files/en-us/web/css/border-inline-start/index.md:48]:
```suggestion
  - : The line style of the border. See {{cssxref("line-style")}}.
```

File: files/en-us/web/css/border-inline/index.md [hunk 5]
```diff
@@ -44,8 +44,8 @@ The `border-inline` is specified with one or more of the following, in any order
 
 - `<'border-width'>`
   - : The width of the border. See {{cssxref("border-width")}}.
-- `<'border-style'>`
-  - : The line style of the border. See {{cssxref("border-style")}}.
+- {{CSSXref("line-style")}}
+  - : The `<line-style>` of the border. See {{cssxref("border-style")}}.
```
Comment (dipikabh) [inline files/en-us/web/css/border-inline/index.md:48]:
```suggestion
  - : The `<line-style>` of the border. See {{cssxref("line-style")}}.
```

**Review by dipikabh — CHANGES_REQUESTED** [review]
Hi @estelle, I think the `<'border-style'>` parameter in the "Values" section is coming from "Formal syntax". We can list that as is in the "Values" section and point it to `<line-style>` enumerated value type in its description, like so:

> - `<'border-style'>`
>   - : The line style of the border. See {{ cssxref("line-style") }}. 

I have indicated this only in a few instances but if you agree, all instances will need to be updated.

File: files/en-us/web/css/-webkit-border-before/index.md [hunk 6]
```diff
@@ -39,10 +41,10 @@ One or more of the following, in any order:
 
 - `<'border-width'>`
   - : See {{cssxref("border-width")}}
-- `<'border-style'>`
+- {{cssxref("line-style")}}
   - : See {{cssxref("border-style")}}
-- `<'color'>`
-  - : See {{cssxref("color")}}
+- {{cssxref("color_value")}}
+  - : See {{cssxref("border-color")}}
```
[the remark this answers is not in the document]
Comment (estelle) [reply files/en-us/web/css/-webkit-border-before/index.md:47]: this would mean the `color` property.

**Comment by estelle** [comment]
When we use `<something>` in css, that something is a data type, not a property. When `<color>` was used, this wasn't a border-color (or border-longhand-color), or column-rule-color property, but rather the [color_value](https://developer.mozilla.org/en-US/docs/Web/CSS/color_value), which is written as the `<color>`  data type.

That's why I changed all the `<'border-style'>` occurrences to `<line-style>` because we are referring to the enumerated data type and not the property.

File: files/en-us/web/css/-webkit-border-before/index.md (hunk 6 above)
[the remark this answers is not in the document]
Comment (dipikabh) [reply files/en-us/web/css/-webkit-border-before/index.md:47]: That's not what I think. From what I understand, listed under the "**Values**" section, `<'color'>` is still a parameter. As per the "[Formal Syntax](https://developer.mozilla.org/en-US/docs/Web/CSS/-webkit-border-before#formal_syntax)":

> -webkit-border-before =
>   <'border-width'> || <'border-style'> || <'color'>

and the [color_value](https://developer.mozilla.org/en-US/docs/Web/CSS/color_value) page defines the `<color>` data type

[the remark this answers is not in the document]
Comment (dipikabh) [reply files/en-us/web/css/-webkit-border-before/index.md:47]: For clarity, can we say:

- `<'color'>`
  - : See [`<color>`](/en-US/docs/Web/CSS/color_value) data type

where `<color>` links to `/en-US/docs/Web/CSS/color_value`

**Comment by dipikabh** [comment]
> When we use `<something>` in css, that something is a data type, not a property. When `<color>` was used, this wasn't a border-color (or border-longhand-color), or column-rule-color property, but rather the [color_value](https://developer.mozilla.org/en-US/docs/Web/CSS/color_value), which is written as the `<color>` data type.

I see. How about we state and format it as in https://github.com/mdn/content/pull/26491#discussion_r1183149282?

> That's why I changed all the `<'border-style'>` occurrences to `<line-style>` because we are referring to the enumerated data type and not the property.

Some are still pending update I guess - like this one https://github.com/mdn/content/pull/26491#discussion_r1182020281?

**Pull request closed without merging**